imgee/templates/index.html.jinja2

```diff
@@ -1,35 +1,25 @@
 {% extends "layout.html.jinja2" %}
-{% block titletags %}
-  <title>{% block title %}{{ config['SITE_TITLE'] }}{% endblock %}</title>
```

Comment:
We should drop use of `config['SITE_TITLE']` now as it interferes with i18n. This code is no longer meant for others to run their own installations.